### DISCUSSION TITLE
Redash

### DISCUSSION THREAD

**Discussion by ravikatariyaunzer** [getredash/redash#6853] [Q&A] [answered]
We are going to use redash in our organisation and would like to know if we are releasing new version frequently and support on community ?

**Comment by justinclift** [comment, marked as answer]
We switched to being a Community Led project some time ago, but haven't created a new official release yet.

As of this month (April) though, we have an automatically updated `preview` image on Docker Hub that's pretty good.

We still need to do more testing before we're satisfied it's stable enough to be "production quality", but some places do already use it in production.

You can try out the `preview` image by using the setup script here along with the `--preview` option, if that helps: https://github.com/getredash/setup/

```
$ sudo ./setup.sh --preview
```

For trying it out, try to ensure that the computer has at least 8GB of ram.  It can go pretty slowly if you use less, but with 8GB it's pretty good. :smile:
